main.go

```diff
@@ -26,8 +28,9 @@ func (c *CLI) Run() error {
 
 	return function.Serve(
 		&Function{
-			log:  log,
-			fsys: &osFS{},
+			log:           log,
+			fsys:          &osFS{},
+			defaultSource: os.Getenv("FUNCTION_GO_TEMPLATING_DEFAULT_SOURCE"),
```

Comment:
awesome, i like this approach of resolving the value in main.go and passing it into the function more than a global var, thanks for that consideration!

now i'm also wondering if this should be exposed as a command line arg, in addition to the env var, similar to `TLSCertsDir`. i'm not sure if it should be exposed at that level also, but it's something else to consider here 🤔

Reply:
I think it's probably cleaner to make it a command line argument with an env var like TLSCertsDir, since it needs to be rebased anyway?